critiquebrainz/db/review.py

```diff
@@ -415,10 +415,11 @@ def list_reviews(*, inc_drafts=False, inc_hidden=False, entity_id=None,
                SUM(
                    CASE WHEN vote = 't' THEN 1
                    WHEN vote = 'f' THEN -1 WHEN vote IS NULL THEN 0 END
-               ) AS rating,
+               ) AS popularity,
                latest_revision.id as latest_revision_id,
                latest_revision.timestamp as latest_revision_timestamp,
                latest_revision.text as text,
+               latest_revision.rating as rating,
```

Comment:
The changes in #125 are dependent on this PR. So its getting a bit confusing. That's the reason I asked you if I could do it in that PR as a seperate commit.